internal/engine/buildcontroller.go

```diff
@@ -76,7 +77,14 @@ func (c *BuildController) needsBuild(ctx context.Context, st *store.Store) (buil
 	}, true
 }
 
+func (c *BuildController) disableForTesting() {
```

Comment:
I guess this isn't necessary if we're just doing this in testing from the engine package.